docs/standard_transaction.md

```diff
@@ -0,0 +1,153 @@
+---
+id: standard_transaction
+title: 6 - The Standard Transaction
+---
+
+You should now be well versed in a number of ways to lock up a coin using a Chialisp puzzle.  We have all the tools we need now to talk about the standard transaction format on the Chia network.
+
+Before you go through this section, it may be worth it to check out this [blog post](https://www.chia.net/2021/05/27/Agrgregated-Sigs-Taproot-Graftroot.html) by Bram Cohen on why the standard transaction is the way it is.
+
+## Pay to "Delegated Puzzle" or "Hidden Puzzle"
+
+If you remember from [part 2](/docs/coins_spends_and_wallets) we created a puzzle that paid to a "delegated puzzle": a puzzle that allows the solver to pass in a puzzle and solution to create their own conditions for the output.  This is one half of the functionality we want our standard transaction to have.
```

Comment:
if we ever want to re-arrange the chapters, calling out "part 2" by number instead of name might be risky